esp/src/src-react/components/forms/landing-zone/XmlImportForm.tsx

```diff
@@ -96,23 +98,42 @@ export const XmlImportForm: React.FunctionComponent<XmlImportFormProps> = ({
                     ) ? "::" : "") + file.TargetName;
                     request["sourceRowTag"] = file.TargetRowTag;
                     request["destNumParts"] = file.NumParts;
-                    FileSpray.SprayVariable({
+                    requests.push(FileSpray.SprayVariable({
                         request: request
-                    }).then((response) => {
+                    }));
```

Comment:
Error reporting

Reply:
Added a .catch(...) a few lines below this comment, after the `Promise.all(requests).then(...)` in each of these forms.